src/config/errorHandlerMiddeware.js

```diff
@@ -0,0 +1,9 @@
+import { isRejectedWithValue } from "@reduxjs/toolkit"
+import { toast } from "react-toastify"
+
+export const rtkQueryErrorLogger = () => (next) => (action) => {
+  if (isRejectedWithValue(action)) {
```

Comment:
We can extend a little our error handling here. 
There is also `isRejected` ( https://redux-toolkit.js.org/api/matching-utilities#isrejected ) in rtk-query, which indicates if promise was rejected. The one we use is only when promise is rejected with value. 

I propose next, we leave `isRejectedWithValue` as it is and we add additional check with `isRejected` down below and we show some useful info like what url, what status code and what error are in these case ( if there are any )

So if we for example will get rejected promise with `data.message` : "No Id found", we will display this. 
If we just get rejected promise without any value, we will anyway show some general error, request url and status code in this case.

To test it, we can try to modify some request on backend to reject without any body.